kubernetes_deploy/development/service.yml

```diff
@@ -11,3 +11,20 @@ spec:
     nodePort: 31000
   selector:
     app: laa-legal-adviser-api
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: laa-legal-adviser-api-mq
+spec:
+  selector:
+    app: laa-legal-adviser-api
+  ports:
+    - protocol: TCP
+      name: mq-5671
+      port: 5671
+      targetPort: 5671
```

Comment:
🙋‍♂ Technically, we don't need 5671 port as it's used for AMQPS (TLS/SSL encrypted AMQP).